### PR TITLE
15 remove gui to complement cli

### LINKED ISSUE #15: Remove GUI to complement CLI
Develop a simple flask app that can be used as an alternative to the CLI for execution

### PR DESCRIPTION
**Pull request by Bullkn0x**
Removed the web app and added to gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ outputs/
 test_files/
 config.json
 *.pem
+web/
```
Comment (LorenDavie) [inline .gitignore:8]: Why are we .gitignoring the web directory?  Removing it should be sufficient.

**Review by LorenDavie — CHANGES_REQUESTED** [review]
.gitignoring the web directory doesn't seem to have a purpose.

File: .gitignore (hunk 1 above)
Comment (Bullkn0x) [reply .gitignore:8]: I have the web directory in my local project to keep if ever needed in the future

Comment (LorenDavie) [reply .gitignore:8]: Make a branch for that - shouldn't go into the code.

**Pull request merged**